macosx/Controller.mm

```diff
@@ -813,8 +813,17 @@ - (void)awakeFromNib
 
     [self updateMainWindow];
 
+    if (@available(macOS 26.0, *))
+        ;
+    else
+    {
+        // <#7908> Keep older macOS clean of visual noise
+        for (NSMenuItem* item in _fWindow.menu.itemArray)
+            for (NSMenuItem* subItem in item.submenu.itemArray)
+                subItem.image = nil;
+    }
+
     //timer to update the interface every second
-    [self updateUI];
```

Comment:
This change looks unrelated.

Reply:
It's already done one line above in `[self updateMainWindow];`